waspc/cli/src/Wasp/Cli/Command/Db/Seed.hs

```diff
@@ -36,7 +36,7 @@ seed maybeUserProvidedSeedName = do
   cliSendMessageC $ Msg.Start $ "Running database seed " <> nameOfSeedToRun <> "..."
 
   liftIO (dbSeed genProjectDir nameOfSeedToRun) >>= \case
-    Left errorMsg -> cliSendMessageC $ Msg.Failure "Database seeding failed" errorMsg
```

Comment:
This is where the old code was swallowing errors. We'd just output a failure message and continue as if nothing happened.